flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/JobExceptionsInfo.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.rest.messages;
+
+import org.apache.flink.api.common.JobID;
+import org.apache.flink.runtime.rest.handler.job.JobExceptionsHandler;
+import org.apache.flink.runtime.rest.util.RestMapperUtils;
+import org.apache.flink.util.Preconditions;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+import javax.annotation.Nullable;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Response type of the {@link JobExceptionsHandler}.
+ */
+@JsonSerialize(using = JobExceptionsInfo.Serializer.class)
+@JsonDeserialize(using = JobExceptionsInfo.Deserializer.class)
+public class JobExceptionsInfo implements ResponseBody {
+
+	public static final String FIELD_NAME_JOB_ID = "jid";
+	public static final String FIELD_NAME_JOB_NAME = "name";
+	public static final String FIELD_NAME_EXECUTION_EXCEPTIONS = "execution-exceptions";
+
+	private final JobID jobId;
+
+	private final String jobName;
+
+	@Nullable
+	private final ExecutionExceptionsInfo executionExceptionsInfo;
+
+	public JobExceptionsInfo(JobID jobId, String jobName, @Nullable ExecutionExceptionsInfo executionExceptionsInfo) {
```

Comment:
This class doesn't appear to resemble the response that the existing JobExceptionsHandler returns. The ExecutionExceptionsInfo contains everything we need.

What is this class for?